src/sections/feeds.ts

```diff
@@ -167,6 +169,7 @@ const GetFeedSubmissionResultResponse = Codec.custom<string>({
 
 export interface GetFeedSubmissionResultParameters {
   FeedSubmissionId: string
+  format?: string
```

Comment:
Any reason why not to use literal types here, which would help to get intellisense and avoid typos?

```
format?: 'json' | 'xml'
```

Reply:
Oh good point! Oversight on my part.